snuba/datasets/storages/processors/replaced_groups.py

```diff
@@ -124,12 +125,11 @@ def process_query(self, query: Query, request_settings: RequestSettings) -> None
                         "processor": str(condition_to_add is not None),
                     },
                 )
-                log_issue = True
-
-            if log_issue:
                 logger.warning(
-                    "Discrepancy between project_extension and replacement consistency processor",
+                    "Groups discrepancy between project_extension and processor.",
+                    extra={
+                        "extension": existing_groups_conditions,
+                        "processor": condition_to_add,
+                    },
```

Comment:
Not sure if this goes to Kibana, but @JTCunning yelled at me and Dan for putting non-scalars in logging yesterday.

Reply:
@JTCunning ? 
I can easily turn them into strings.

Reply:
You're good, here. This goes into StackDriver Logging. I'd actually like to see how they handle some of these more complex non-scalar values. It's only the main Sentry product that goes into the EFK stack.